app/views/categories/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="container">
+  <%= form_with model: @category, method:'post', class:'form',url: categories_path do |form| %>
+  <div class="flex main-bg-color color-white justify-center align-center justify-between height-40px">
+    <%= link_to "🡄", categories_path, method: :get, class:"back ml-10" %>
+    <h2>Add Category</h2>
+    <div class="actions color-white mr-10">
+      <%= form.submit "Create" %>
+    </div>
+  </div>
+    <div class="field">
+      <div class="flex row align-center margin-10">
+        <img src=<%=image_path("save-money.png")%>/>
+        <label id="label-for-image" for="category_image" 
+            class="text-color font-large margin-left-10 underlined">
+          Upload picture 
+        </label>
+        <%= form.file_field :image, class:"inputfile" %>
+      </div>
+      <%= form.text_field :name, autofocus: true, placeholder: "Name" %>
+    </div>
+  <% end %>
+</div>
+
+<script>
+  document.getElementById('category_image').addEventListener('change', (e) => {
+    if(e.target.files.length == 1){
+      fileName = e.target.files[0].name;
+      document.getElementById('label-for-image').textContent = fileName;
+    }
+    console.log(e.target.files)
+  });
```

Comment:
- [ ] One of the major requirements for this project is that a user should be able to indicate the `name` of a category and the `icon` they wish to use when creating a category. Currently, the user can't indicate the icon to be used. I get this error whenever I try to create a category with an icon:

![image](https://user-images.githubusercontent.com/77106962/176554853-983806d7-82fb-43e4-aa19-3832938f81c7.png)

#### Kindly make it possible for the user to indicate the `icon` to be used when creating a category. 👍